dash_docs/chapters/advanced_callbacks/index.py

```diff
@@ -89,5 +89,72 @@ def slow_function(input):
     The [Performance](/performance) section of the Dash docs delves a
     little deeper into leveraging multiple processes and threads in
     conjunction with memoization to further improve performance.
-    ''')
+    '''),
+
+    rc.Markdown('''
+    ## When Are Callbacks Executed?
+
+    This section describes the circumstances under which the `dash-renderer` front-end client can make a request to the Dash back-end server to execute a callback function.
```

Comment:
```suggestion
    This section describes the circumstances under which the `dash-renderer` front-end client can make a request to the Dash back-end server (or the clientside callback code) to execute a callback function.
```
just to make it clear that all of this applies exactly the same way to clientside callbacks.